src/Dfe.PlanTech.Domain/Dfe.PlanTech.Domain.csproj

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="contentful.csharp" Version="8.3.0" />

```

Comment:
howcome this needed adding to the domain project?

Reply:
We use it in the `ContentBase` model which I've added to the domain project

`public class ContentBase : Contentful.Core.Models.Entry<ContentBase>`

Reply:
ahhh ok - makes sense